.env

```diff
@@ -0,0 +1,9 @@
+MYSQL_ROOT_USERNAME=root
+MYSQL_ROOT_PASSWORD=bookPass
```

Comment:
add .env to gitignore
remove it from here with git rm --cached .env
add .env.template with actual values removed